Decred Wallet/Features/Security/RequestPinViewController.swift

```diff
@@ -17,95 +17,112 @@ class RequestPinViewController: SecurityBaseViewController {
     var onUserEnteredPin: ((_ pin: String) -> Void)?
     
     var requestPinConfirmation = false
-    var pinToConfirm: String = ""
     
-    @IBOutlet weak var cancelBtn: UIButton!
+    var submitEnabled: Bool = false{
+        didSet{
+            if self.submitEnabled == true{
+                self.btnCommit.isEnabled = true
+                btnCommit.setBackgroundColor(UIColor.appColors.decredBlue, for: .normal)
+            }else{
+                self.btnCommit.isEnabled = false
+                btnCommit.setBackgroundColor(UIColor.appColors.darkGray, for: .normal)
+            }
+        }
+    }
+
+    @IBOutlet weak var btnCommit: UIButton!
+    
+    // PIN Inputs
```

Comment:
Comment is not necessary. Variable names are/should be descriptive enough. This applies to similar comments below.

Reply:
resolved in 50488de